mne/preprocessing/artifact_detection.py

```diff
@@ -23,8 +23,8 @@ def annotate_movement(raw, pos, rotation_velocity_limit=None,
 
     Parameters
     ----------
-    raw instance : instance of Raw. Data to detect head movement for.
-        Instance of a raw.
+    raw instance : instance of Raw.
```

Comment:
just `raw : instance of Raw` not `raw instance : ...`. The text on the left of `:` is the variable name, and variable name is `raw`.